wic/src/components/ResoruceMap.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes, Component} from 'react';
+import ReactDOM from 'react-dom';
+
+let map;
+
+export default class ResoruceMap extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const geocode_options = {
+      address: this.props.zipcode || '94010'
+    }
+    new window.google.maps.Geocoder().geocode(geocode_options, (results, status) => {
+      if (status == 'OK') {
```

Comment:
IMO: its best to use triple-equals, otherwise, JavaScript does some "magic" with coercion and...kind be odd.

Reply:
replaced with string.test